refactor/urls.js

```diff
@@ -1,16 +1,37 @@
-// Refactor this code
+function getURL(dev = false, tasks = {}, additionalParams = {}) {
+  const BASE_URL = "/tasks";
+  const DEV_PARAMS = {
+    status: "ACTIVE",
+    dev: "true",
+    size: "20",
+  };
 
-function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+  tasks = tasks || {};
 
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
-    }
+  const params = new URLSearchParams();
 
-    if (tasks.prevTasks) {
-        url = '/tasks?hasPrev=true';
-    }
-    return { url };
+  if (dev) {
+    Object?.entries(DEV_PARAMS)?.forEach(([key, value]) => {
```

Comment:
Not. Do we need `Object?` JS standard should guarantee it. (and if really only JS, then ?. optional chaining operator would also not work anyway)